bin/run

```diff
@@ -64,7 +60,7 @@ if not model_path.is_file() and config.model_url is not '':
 tf.reset_default_graph()
 
 input_spec = []
-with gfile.FastGFile(model_file, 'rb') as f:
+with gfile.GFile(model_file, 'rb') as f:
```

Comment:
Why did you decide to slow down gfile? 😅

Reply:
It was deprecated! Assume they defaulted to fast haha.